CONTRIBUTING.md

```diff
@@ -74,25 +74,27 @@ The test suite runs with `py.test`. The following are some useful commands for r
 tests, assuming you are in the `conda-build` root folder:
 
 ### Run all tests:
-```
-    py.test tests
+```bash
+    make test
+    # Or if you prefer pytest directly:
+    py.test
```

Comment:
```suggestion
```bash
    # On Linux and Mac OS X
    make test
```

```bash
    # Or if you prefer running pytest directly:
    py.test
```